.github/workflows/tests.yml

```diff
@@ -181,20 +181,21 @@ jobs:
         wget "https://github.com/NationalSecurityAgency/ghidra/releases/download/Ghidra_${{ matrix.ghidra-version }}_build/ghidra_${{ matrix.ghidra-version }}_${{ matrix.public-version }}.zip" -O ./.github/ghidra/ghidra_${{ matrix.ghidra-version }}_PUBLIC.zip
         unzip .github/ghidra/ghidra_${{ matrix.ghidra-version }}_PUBLIC.zip -d .github/ghidra/
     - name: Install Ghidrathon
-      run : |
+      run : | # Colton TODO: fix wget url & unzip paths pending merge of breaking Ghidrathon changes
```

Comment:
I have released https://github.com/mandiant/Ghidrathon/releases/tag/v3.0.0 with these breaking changes. Let's update the code here to reflect that.